pandas/tests/frame/test_to_csv.py

```diff
@@ -1203,3 +1203,16 @@ def test_period_index_date_overflow(self):
 
         expected = ',0\n1990-01-01,4\n,5\n3005-01-01,6\n'
         assert result == expected
+
+    def test_multi_index_header(self):
+        # see gh-5539
+        columns = pd.MultiIndex.from_tuples([("a", 1), ("a", 2),
+                                             ("b", 1), ("b", 2)])
+        df = pd.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8]])
```

Comment:
Looks like `DataFrame` and `MultiIndex` are already imported, so could remove the `pd.`, if you feel that's cleaner/worth an edit.